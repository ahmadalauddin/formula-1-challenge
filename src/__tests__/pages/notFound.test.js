import  { shallow } from 'enzyme';

import PageNotFound from '../../pages/notFound';


let wrapper;

beforeEach(()=>{
  wrapper = shallow(<PageNotFound/>);
})


it("Checking snapshots", ()=>{
  expect(wrapper).toMatchSnapshot();
});

it("Checking if padding is applied on top", ()=>{
  expect(wrapper.exists(".pt-88")).toEqual(true);
});

