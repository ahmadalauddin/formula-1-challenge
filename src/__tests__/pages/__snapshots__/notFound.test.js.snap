// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking snapshots 1`] = `
<div
  className="pt-88"
>
  <Page404 />
</div>
`;
